endpoints/api/test/test_immutability_policy.py

```
@@ -0,0 +1,213 @@
+    OrgImmutabilityPolicies,
+    OrgImmutabilityPolicy,
+    RepositoryImmutabilityPolicies,
+    RepositoryImmutabilityPolicy,
+)
+from endpoints.api.test.shared import conduct_api_call
+from endpoints.test.shared import client_with_identity
+from test.fixtures import *  # noqa: F401, F403
+
+
+class TestOrgImmutabilityPolicies:
+    def test_list_empty(self, initialized_db, app):
```

Ruff (0.14.13)

Unused method argument: `initialized_db`

(ARG002)